study-builder/studies/osteo/study.conf

```diff
@@ -1631,6 +1646,20 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "FAMILY_HISTORY"
```

Comment:
There's also a little Thank You page after completing the family history survey. How are we thinking about addressing that? I think one possibility to try is to use "workflows transitions": once activity is complete, client will ask where to go next, the workflow transitions rule will then say "thank you" page is next, and then frontend will have a static family-history-thank-you page that it will show.

Reply:
As we discussed on Slack, I have added a new workflow translation for the thank you page:
191bb957a165da700f91785495d6df7f950600cf
I also created a new Angular ticket to create the page and implement the proper thank you page redirect.
https://broadinstitute.atlassian.net/browse/DDP-5673